README.md

```diff
@@ -345,6 +345,16 @@ the attribute type does not have any special semantic or effect currently.
 As already mentioned, all this configuration parameters can also be adjusted using the
 [`config.js`](https://github.com/telefonicaid/IoT-STH/blob/develop/config.js) file whose contents are self-explanatory.
 
+It is important to note that there is a limitation on the bytes a MongoDB namespace (concatenation of the database name and
```

Comment:
You are right, I did not notice the switch, I only saw the last case... Tip: do not review PRs with fever xD NTC